### PR TITLE
prow: add metal3-io/.github to prow

### PR DESCRIPTION
**Pull request by tuminoid**
Activate prow for the .github special repository.

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/lgtm

File: prow/config/config.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ tide:
     Nordix/metal3-clusterapi-docs: rebase
   queries:
     - repos:
+        - metal3-io/.github
```
Comment (furkatgofurov7) [inline prow/config/config.yaml:39]: this is not correct actually (should be metal3-io/metal3-io.github.io), besides it is already there (line 49)

File: prow/config/plugins.yaml [hunk 2]
```diff
@@ -51,6 +51,7 @@ plugins:
       - wip
       - yuks
       - require-matching-label
+  metal3-io/.github:
```
Comment (furkatgofurov7) [inline prow/config/plugins.yaml:54]: same

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
/hold

File: prow/config/config.yaml (hunk 1 above)
Comment (tuminoid) [reply prow/config/config.yaml:39]: No, `.github` is special repository on the organization level, it is not the same as the book :)

Comment (kashifest) [reply prow/config/config.yaml:39]: its different @furkatgofurov7 , we have added also .github repo

Comment (lentzi90) [reply prow/config/config.yaml:39]: Both repos exist actually.
- https://github.com/metal3-io/metal3-io.github.io
- https://github.com/metal3-io/.github

**Review by kashifest — APPROVED** [review]

**Review by furkatgofurov7 — COMMENTED** [review]
Was not aware of the new repo, all should be good.

/hold cancel

File: prow/config/config.yaml (hunk 1 above)
Comment (furkatgofurov7) [reply prow/config/config.yaml:39]: Thanks @kashifest @lentzi90

**Pull request merged**